student.rb

```diff
@@ -1,4 +1,6 @@
+# student.rb
 require_relative 'person'
+require_relative 'classroom'
 
 # Class representing a student, inheriting from Person
 class Student < Person
```

Comment:
- [ ] 🌟 Just like with the Classroom's class, let's aim to make the classroom class read-only in your student class. Please use `attr_reader :classroom` instead of `attr_accessor :classroom`. This way, the classroom remains read-only! 😊📚